README.md

```diff
@@ -67,3 +67,18 @@ $ ansible-playbook my-ansible-playbook.yml -i my-ansible-inventory.yml
 ```
 ## Running tests
 To run unit and integration tests, run "pytest" in a terminal.
+
+## Running in Docker
+To build a docker image use command
+```bash
+$ docker build --target production --tag todo-app:prod .
+```
+
+To run image use command
+```bash
+$ docker run --env-file ./.env --publish 80:8000 todo-app:prod .
```

Comment:
I don't think this final `.` is doing anything here - it's a vital (and easily forgotten!) argument for the `docker build ...` command, but it's just being sent to Gunicorn/Flask as an extra argument that does nothing here